### PR TITLE
Update badge for chat that ha moved from Gitter to Zulip

### PR DESCRIPTION
**Pull request by apupier**
Signed-off-by: Aurélien Pupier <apupier@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 [![Build Status](https://travis-ci.com/camel-tooling/vscode-camelk.svg?branch=master)](https://travis-ci.com/camel-tooling/vscode-camelk)
 [![CircleCI](https://circleci.com/gh/camel-tooling/vscode-camelk.svg?style=shield)](https://circleci.com/gh/camel-tooling/vscode-camelk)
 [![License](https://img.shields.io/badge/license-Apache%202-blue.svg)]()
-[![Gitter](https://img.shields.io/gitter/room/camel-tooling/Lobby.js.svg)](https://gitter.im/camel-tooling/Lobby)
+[![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://camel.zulipchat.com/#narrow/stream/258729-camel-tooling)
```
Comment (lhein) [inline README.md:5]: Would just "Chat" be a better name?

Comment (bfitzpat) [reply README.md:5]: I agree with Lars here

**Review by bfitzpat — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (bfitzpat) [reply README.md:5]: or even just "Zulip" or "Zulip Chat" to match the old "Gitter" link

Comment (apupier) [reply README.md]: updated to "Zulip chat"

**Review by lhein — APPROVED** [review]

**Pull request merged**